Integrations/src/main/java/io/deephaven/integrations/learn/Gatherer.java

```diff
@@ -1,30 +1,55 @@
 package io.deephaven.integrations.learn;
 
 import io.deephaven.db.v2.sources.ColumnSource;
+// REMOVE THIS AFTER REMOVING makeIndexSet!
+import io.deephaven.db.tables.Table;
 
+/**
+ * Gatherer takes Deephaven columnar data and places it into a buffer to be used by Python.  The Python object will take
+ * data from the buffer and use it to construct a 2d array of specified size.
+ */
 public class Gatherer {
 
+    /**
+     * Generate an index set from a table.
+     * THIS IS JUST FOR EASY PYTHON TESTING.  REMOVE BEFORE PROD.
+     *
+     * @param t table
+     * @return index set
+     */
+    public static IndexSet makeIndexSet(final Table t) {
+        final IndexSet is = new IndexSet(t.intSize());
+
+        for(long idx : t.getIndex()) {
+            is.add(idx);
+        }
+
+        return is;
+    }
+
     /**
      * Copy data from a table into a 2d tensor of Booleans.
      *
      * @param indexSet indices of the rows of the table to put into the tensor
      * @param columnSources columns of data to put into the tensor
+     * @param transpose a boolean indicating the user's intent to return a transposed Python object
```

Comment:
```suggestion
     * @param transpose true to return a tensor buffer in transposed order; false to return the tensor buffer in regular order.
```
Apply the change everywhere.
1) Javadoc knows the types, so you don't need that in doc strings.
2) Every parameter is expressing the user's intent, so that does not need to be said.
3) No code here is python specific.  We're just making a general memory buffer.